cachi2/core/models/output.py

```diff
@@ -83,11 +82,18 @@ def unique_packages(cls, packages: list[Package]) -> list[Package]:
         )
 
     @pydantic.validator("environment_variables")
-    def unique_env_vars(cls, env_vars: list[EnvironmentVariable]) -> list[EnvironmentVariable]:
+    def _unique_env_vars(cls, env_vars: list[EnvironmentVariable]) -> list[EnvironmentVariable]:
         """Sort and de-duplicate environment variables by name."""
         return unique_sorted(env_vars, by=lambda env_var: env_var.name)
 
     @classmethod
     def empty(cls):
         """Return an empty RequestOutput."""
         return cls(packages=[], environment_variables=[])
+
+
+def merge_outputs(outputs: Iterable[RequestOutput]) -> RequestOutput:
```

Comment:
Does it make sense to move this to the `resolver.py` module?

Reply:
Moved and dropped the unit test, it seemed like overkill for a function this simple anyway

Reply:
I tend to like small, focused functions, anyway :sweat_smile: 
I'd argue that if they're private, they don't necessarily need to be unit tested separately, though.